pymarl/learners/ddmac_learner.py

```diff
@@ -51,7 +51,8 @@ def train(self, batch: EpisodeBatch, t_env: int, episode_num: int):
         for t in range(batch.max_seq_length - 1):
             agent_outs = self.mac.forward(batch, t=t)
             current_behavior_out.append(agent_outs)
-        current_behavior_out = th.stack(current_behavior_out, dim=1)
+        current_behavior_out = th.stack(current_behavior_out, dim=1)  # Concat over time
+        mac_out = current_behavior_out.clone()
```

Comment:
Great! I am sure it will run faster now.

Reply:
Let's hope it works as good as before, but I have no doubt :D